core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6298,6 +6298,22 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_UFS_ENABLED =
+      booleanBuilder(Name.USER_UFS_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("If this is enabled, this client will talks to the target UFS directly "
+              + "without the interaction with Alluxio servers.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
+          .setScope(Scope.CLIENT)
+          .build();
+  public static final PropertyKey USER_UFS_ADDRESS =
```

Comment:
is it the address to the namenode?

Reply:
this is the full UFS address e.g. hdfs://namenode:port/path, s3://bucket/path etc, Similar value as the MASTER_MOUNT_TABLE_ROOT_UFS

Reply:
is it better to use USER_UFS_ROOT_PATH ? your call

Reply:
This will be modified and merged on https://github.com/Alluxio/alluxio/pull/16292